GVRf/Framework/src/org/gearvrf/GVRCameraRig.java

```diff
@@ -156,6 +159,8 @@ public float getFloat(String key) {
      *            The {@code float} value to map.
      */
     public void setFloat(String key, float value) {
+        checkStringNotNullOrEmpty("key", key);
+        checkFloatNotNaNOrInfinity("Float value for shader", value);
```

Comment:
`"Float value for shader"` s/b just `"value"`: A literal parameter name, not any sort of description.